db/migrate/20220121100200_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :bio
+      t.integer :postcounter
```

Comment:
- [ ] Following ruby best practice for naming convention. It will be nice to use snake_case for methods and variables. 

replace `:postcounter` with `:post_counter`

Reply:
This issue is well resolved! ✔️